nipype/pipeline/engine/nodes.py

```diff
@@ -439,7 +443,8 @@ def run(self, updatehash=False):
         )
 
         # Check hash, check whether run should be enforced
-        logger.info('[Node] Setting-up "%s" in "%s".', self.fullname, outdir)
+        if not isinstance(self, MapNode):
```

Comment:
Because the log trace used to show up for every subnode in the map (plus the initial one from `Node`). Now `Node` and `MapNode` only show one "setting up" trace.